Lib/test/test_robotparser.py

```diff
@@ -96,17 +96,31 @@ class RejectAllRobotsTest(BaseRobotTest, unittest.TestCase):
     bad = ['/cyberworld/map/index.html', '/', '/tmp/']
 
 
+class EmptyFileTest(BaseRobotTest, unittest.TestCase):
+    robots_txt = ""
+
+
+class NoRequestRateAndCrawlDelayTest(BaseRobotTest, unittest.TestCase):
```

Comment:
I believe this case is already covered by other tests, e.g. `RejectAllRobotsTest`.